nifi-nar-bundles/nifi-enrich-bundle/nifi-enrich-processors/pom.xml

```diff
@@ -90,6 +89,11 @@
             <version>1.16.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.jr</groupId>
+            <artifactId>jackson-jr-objects</artifactId>
```

Comment:
This dependency appears to be used only in test classes, is that correct? If so, the `scope` should be set to `test`.